### ISSUE TITLE
Update docs for sirepo migration

### ISSUE THREAD

**Issue by robnagler** [radiasoft/jupyter.radiasoft.org#123]

**Comment by robnagler** [comment]
I think the docs need to be updated still.

**Comment by bruhwiler** [comment]
@robnagler -- can this issue be closed?

If not, can you provide some context regarding the docs that need to be updated?

**Comment by robnagler** [comment]
The [README.md](https://github.com/radiasoft/jupyter.radiasoft.org#readme) is out of date. It should talk about how you use sirepo.com/jupyter. Possibly, the repo should be renamed to sirepo-jupyter. Or perhaps, the repo should be archived with a statement on the README to go to rsft.us/github or some other link we own. Someone needs to give thought how to present Jupyter iow. This repo is not the right way.

**Comment by bruhwiler** [comment]
Agreed.  However, that is a separate issue.
In the near term, I want an updated list.

**Comment by bruhwiler** [comment]
I agree.  This is an important conversation for the product management team.
